selenium_tests/course_info/course_info_tests.py

```diff
@@ -0,0 +1,106 @@
+import urlparse
+
+from selenium.common.exceptions import TimeoutException
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support.ui import WebDriverWait
+
+from selenium_tests.course_info.course_info_base_test_case import CourseInfoBaseTestCase
+from selenium_tests.course_info.page_objects.editor_page import EditorPage
+from selenium_tests.course_info.page_objects.main_page import MainPage
+from selenium_tests.pin.page_objects.pin_login_page_object import PinLoginPageObject
+
+
+class CourseInfoTestFlow(CourseInfoBaseTestCase):
```

Comment:
Can we change this to just `CourseInfoTests`? It's not really a flow, and the tests should be mutually exclusive.